ExamPortal/Models/testRecord.js

```diff
@@ -0,0 +1,24 @@
+const mongoose  = require('mongoose')
+
+const Schema = mongoose.Schema
+
+const testSchema = new Schema({
```

Comment:
rename the schema to `candidateAnswer` or suggest a better name, `testSchema` is misleading.